data/card_catalogue/movement.json

```diff
@@ -37,17 +43,23 @@
     "upgrade": "",
     "tier": "gold",
     "details": {
-      "rules": "Teleport the entire party anywhere",
+      "rules": {
+        "active": "Teleport the entire party anywhere.",
+        "passive": ""
+      },
       "flavour": "Endless folds in spacetime reveal themselves to the worthy",
-      "image": "assets/core/card/illustration/Gerraro_s Door.png"
+      "image": "assets/core/card/illustration/Leopold's Door.png"
     }
   },
   {
     "title": "Queen's Gambit",
     "upgrade": "",
     "tier": "red",
     "details": {
-      "rules": "Move instantly as many spaces as you want in a straight line.",
+      "rules": {
+        "active": "Move as far as you want in an unobstructed horizontal line.",
+        "passive": ""
+      },
```

Comment:
instant is implied, added an 'unobstructed' clause

Reply:
Something about `horizontal` doesn't sit right with me, but I can't think of a way around it 🤔

Reply:
I'm gonna move this one to 
- #79